packages/editor-ui/src/Interface.ts

```diff
@@ -272,6 +271,8 @@ export interface INewWorkflowData {
 
 export interface WorkflowMetadata {
 	onboardingId?: string;
+	templateId?: string;
+	instanceId?: string;
```

Comment:
are we adding instance id now to the metadata? we intentionally avoided adding that before when saving.. only to be added when sharing..

Reply:
We are not adding `instanceId` on any new places in this PR, I moved it here so we can re-use this type in `IWorkflowToShare` and clean up the typing around that a bit more.